app/models/chargeback.rb

```diff
@@ -80,7 +80,7 @@ def get_rates(perf)
     parents = [perf.parent_host, perf.parent_ems_cluster, perf.parent_storage, perf.parent_ems, @enterprise].compact
     parents.push(tenant_resource) unless tenant_resource.nil?
 
-    @rates[key] = ChargebackRate.get_assigned_for_target(perf.resource, :tag_list => tag_list, :parents => parents, :associations_preloaded => true)
+    @rates[key] = ChargebackRate.get_assigned_for_target(perf.resource, :tag_list => tag_list, :parents => parents)
```

Comment:
Yes. I hadn't intended on this happening, but most of the preloading requirements here have been removed.

This method in general can probably be removed. Or reworked at the least. It is here to work around the performance problems in `assignment_mixin` - many of which have been fixed.